bittide-instances/src/Bittide/Instances/MVPs.hs

```diff
@@ -148,5 +172,35 @@ stickyBits SNat = mealy go (0 , unpack 0)
       | holdingBits = (storedBits, predCount)
       | otherwise   = (unpack 0, predCount)
 
+ibufds
+  :: (KnownDomain dom)
```

Comment:
Documentation please. You can just give a very terse explanation and refer to the Xilinx docs.